internal/pkg/nats/client.go

```diff
@@ -153,3 +153,11 @@ func (c *Client) Disconnect() error {
 	}
 	return c.connection.Drain()
 }
+
+func (c *Client) PublishBinaryData(data []byte, topic string) error {
+	return nil
+}
+
+func (c *Client) SubscribeBinaryData(topics []types.TopicChannel, messageErrors chan error) error {
+	return nil
```

Comment:
return the error as the zeromq one
`return fmt.Errorf("not supported SubscribeBinaryData func")`

Reply:
Updated.